ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.component.ts

```diff
@@ -179,6 +180,10 @@ export class AddressInputComponent implements OnInit, OnDestroy {
   set addressErrors(addressErrors: AddressError[]) {
     this.displayVerificationErrors(addressErrors);
   }
+
+  @Input()
+  phoneRequired: boolean = false;
```

Comment:
tslint warning that type is redundant. the `false` assignment enough to know it is boolean.